app/models/manageiq/providers/amazon/storage_manager/ebs/cloud_volume.rb

```diff
@@ -100,8 +100,9 @@ def provider_object(connection = nil)
 
   def modify_volume_options(options = {})
     modify_opts = {}
+
     if volume_type != 'standard'
-      modify_opts[:volume_type] = options[:volume_type] if options[:volume_type] && options[:volume_type] != 'standard'
+      modify_opts[:volume_type] = options[:volume_type] if options[:volume_type] && options[:volume_type] != volume_type
       modify_opts[:size]        = Integer(options[:size]) if options[:size] && Integer(options[:size]).gigabytes != size
```

Comment:
@gberginc -Looks like this will allow non-standard volume types to be updated with a 'standard' volume type which wasn't allowed before, is that right?